README.md

```diff
@@ -44,6 +44,30 @@ This command runs a recent pyth-client docker image that already has the necessa
 Therefore, once the container is running, all you have to do is run `cd pyth-client && ./scripts/build.sh`.
 Note that updates to the `pyth-client` directory made inside the docker container will be persisted to the host filesystem (which is probably desirable).
 
+### Local development
```

Comment:
This readme doesn't work on my laptop. Then again I've never been able to build locally, so I see no harm in approving.